apps/backend-e2e/src/maps.e2e-spec.ts

```diff
@@ -3280,4 +3280,107 @@ describe('Maps', () => {
         req.unauthorizedTest('maps/1/reviews', 'get'));
     });
   });
+
+  describe('maps/{mapID}/reviews/{reviewID}', () => {
+    describe('GET', () => {
+      let u1, u2, u2Token, map, review;
+      beforeAll(async () => {
+        [u1, [u2, u2Token]] = await Promise.all([
+          db.createUser({ data: { roles: Role.MAPPER } }),
+          db.createAndLoginUser({ data: { roles: Role.REVIEWER } })
+        ]);
+
+        map = await db.createMap({
+          status: MapStatusNew.PUBLIC_TESTING,
+          submitter: { connect: { id: u1.id } }
+        });
+
+        review = await prisma.mapReview.create({
+          data: {
+            mainText:
+              'Sync speedshot into sync speedshot, what substance did you take mapper?!?!',
+            suggestions: [
+              {
+                track: 1,
+                gamemode: Gamemode.RJ,
+                tier: 2,
+                comment: 'True',
+                gameplayRating: 1
+              }
+            ],
+            mmap: { connect: { id: map.id } },
+            reviewer: { connect: { id: u2.id } },
+            resolved: false
+          }
+        });
+      });
+
+      afterAll(() => db.cleanup('mMap', 'user', 'mapReview'));
+
+      it('should return the requested review', async () => {
+        const response = await req.get({
+          url: `maps/${map.id}/reviews/${review.id}`,
+          status: 200,
+          validate: MapReviewDto,
+          token: u2Token
+        });
+        expect(response.body).toMatchObject({
+          mainText: review.mainText,
+          mapID: map.id,
+          id: review.id
+        });
+      });
+
+      it('should return the requested review including author information', async () => {
+        await req.expandTest({
+          url: `maps/${map.id}/reviews/${review.id}`,
+          token: u2Token,
+          expand: 'reviewer',
+          validate: MapReviewDto,
+          paged: false,
+          expectedPropertyName: 'reviewer'
```

Comment:
This property is unnecessary (I should really write proper docs for these methods lol)